opencti-platform/opencti-graphql/src/modules/attributes/internalObject-registrationAttributes.ts

```diff
@@ -240,6 +240,7 @@ const internalObjectsAttributes: { [k: string]: Array<AttributeDefinition> } = {
       multiple: false,
       upsert: false,
       isFilterable: false,
+      sortBy: { path: 'group_confidence_level.max_confidence', type: 'numeric' },
```

Comment:
I'm not sure to understand how it would work here if we wanted to sort by another path then `group_confidence_level.max_confidence`. I can see in the unit tests that we could sort by `group_confidence_level.overrides.entity_type`, how would we be able to do this with this mapping definition ?

Reply:
The theory has made us firts think that way : being able to define different way of sorting a same object by different path.

However this is not possible right now due to the way the API/client work (sorting criteria are fixed enum values that must correspond to a first level attribute name).
We decided to simply and specify one possible path only for a given object.

>  I can see in the unit tests that we could sort by group_confidence_level.overrides.entity_type, how would we be able to do this with this mapping definition ?

This is only a unit test for the utility function `getAttributeMappingFromPath`. It's a generic purpose function, to recursively find the mapping at a given path in the schema. As said earlier, it would be great if we wanted more complex sorting possibilities, but we will not go into that direction for now.